blossom/templates/app/partials/tutorial.partial

```diff
@@ -1,37 +1,356 @@
+                    can do to it! Pan around, zoom in, zoom out, and make it full screen
+                    if you want so that you can find all the tiniest details!
+                </p>
+                <div class="row">
+                    <div class="col-0 col-lg-2"></div>
+                    <div class="col-12 col-lg-8">
+                        <div class="text-center">
+                            <img src="{% static 'images/tutorial_3.png' %}" style="width: 100%"
+                                 alt="A screenshot showing the original T-Rex image, but this time in a different frame with some buttons underneath."
+                            >
+                        </div>
+                    </div>
+                    <div class="col-0 col-lg-2"></div>
+                </div>
+
+                <p class="mt-3">
+                    If you don't like where the image is (or you zoomed in too far and
+                    lost it) then use the controls at the top of the image (not visible
+                    here) to reset it.
+                </p>
+                <p>
+                    You also have access to a new button here: the
+                    <strong>View Rules</strong> button. This will show you all the rules
+                    relating to the place where this image came from; it's on you to
+                    identify whether this image breaks the rules or not! If it doesn't,
+                    then it's good to go... but if it does, you'll want to report it so
+                    that it can be removed. More on that shortly!
+                </p>
+            </div>
+            <div class="modal-footer">
+                <button class="btn btn-primary" data-bs-target="#tutModalToggle3" data-bs-toggle="modal"
+                        data-bs-dismiss="modal">Back
+                </button>
+                <button class="btn btn-primary" data-bs-target="#tutModalToggle5" data-bs-toggle="modal"
+                        data-bs-dismiss="modal">Next
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="modal fade" id="tutModalToggle5" aria-hidden="true" aria-labelledby="tutModalToggleLabel5"
+     tabindex="-1">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="tutModalToggleLabel5">Tutorial: Transcribing</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <p>
+                    Here is where the magic happens. Just write down what you see in the
+                    image! Remember that the goal here is that someeone who cannot access
+                    the image should be able to understand what is going on just by reading
+                    what you've written.
```

Comment:
Maybe we should mention in some way that we use Markdown formatting?

But I'm not sure how to best introduce it, a Markdown tutorial would definitely be too big for this